governance/delegates/0xAF8aa6846539033Eaf0c3ca4C9C7373e370E039b/profile.md

```diff
@@ -1,52 +1,45 @@
 ---
-name: Template Delegate
-url: https://forum.makerdao.com/u/example
-profile_picture_url: https://images.com/image.jpeg
+name: Flip Flop Flap Delegate LLC 
+url: https://forum.makerdao.com/u/elprogreso/
+profile_picture_url: https://imgur.com/a/l9dkYAN
 ---
 
 # Delegate Name
 
 ### Key Info
 
-Delegate Address: 0xEtcetera  
-Forum: @LongForWisdom  
-Rocketchat: @LongForWisdom  
-Email (optional): longforwisdomandstuff@gmail.com  
-External Website (optional): www.longforwisdom.com  
-Meet Your Delegate Video (optional): youtube or something  
+Delegate Address: flapper.eth  
```

Comment:
I don't think this points to your contract?

Reply:
It should point now to my contract